packages/react/src/themes/teams/components/Dropdown/dropdownStyles.ts

```diff
@@ -144,17 +147,14 @@ const dropdownStyles: ComponentSlotStylesInput<DropdownPropsAndState, DropdownVa
 
   list: ({ props: p, variables: v }): ICSSInJSStyle => ({
     outline: 0,
-    position: 'absolute',
-    borderRadius: v.listBorderRadius,
+    borderRadius: getOpenBorderStyles(p.position === 'below' ? 'top' : 'bottom', v),
```

Comment:
Can we omit this function and write it more flat, but more obvious?

```js
...p.open && p.position === 'top' && ({ borderBottomRaduis: v.borderBottomRadius }),
...p.open && p.position === 'bottom' && ({ borderBottomRaduis: v.borderBottomRadius }),
// etc
```

It's much more readable, see ##1301.

Reply:
done, thanks